### PR TITLE
UI enhancement

### LINKED ISSUE #242: 🪲[Bug]: Smaller screens
**Describe the bug**
The page when for smaller dimensions the side scroller is being his and all other make the spot. I will make sure to align it properly.

**Screenshots**
Screenshot_20240520-232241

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by sharmi4590**
## Description

Hey @Anishkagupta04  

## Related Issues

- Closes #242 
I have made the home page responsive.
## Type of PR

- [x] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/assets/102676949/4108b9a5-9f88-4d25-a48f-9dd16aca10a0

## Checklist

- [x] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have performed a self-review of my code
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,8 +41,54 @@
...
+        display: inline-block;
+        background-color: #55a5ea;
+        color: #fff;
+        padding: 10px 20px;
+        text-decoration: none;
+        border-radius: 5px;
+        margin-top: 20px;
+    }
+
+    @media screen and (max-width: 768px) {
```
Comment (varshith257) [inline index.html:76]: I think a standard mobile screen max-width could be 530 px. Can you add this media screen too?

**Comment by varshith257** [comment]
@sharmi4590 Follow up with the above review comments

**Comment by sharmi4590** [comment]
@varshith257 The code will be working for standard mobile screen as well.
Screenshot (474)

**Comment by Anishkagupta04** [comment]
The phone view is removed from the right side the logo has been added there why have you removed that

**Comment by sharmi4590** [comment]
https://69glitch.github.io/RapiDoc/ In this link the video was at the side of the text in home page

Screenshot (492)

whereas while cloning the project , the home page had a logo at the side , so to make it same i added the video.
Do you want me to make it responsive with the logo only and not the video?

Screenshot (493)

**Comment by varshith257** [comment]
@sharmi4590 Yes, pull the latest upstream and look for changes made recently and follow up here on the progress

**Comment by sharmi4590** [comment]
@varshith257 , made the changes according to your requirements.
https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/assets/102676949/b5ae9d34-a294-4f7f-b0f8-d35e6366e25f

**Comment by sharmi4590** [comment]
@Anishkagupta04 , Please assign the gssoc label and level label .
Thank you.

**Pull request merged**